### PR TITLE
Navigation super squirrel

### PR DESCRIPTION
**Pull request by emilyville**
- put patient info on left side
- fix top nav highlighting
- put patient controls in button dropdown
- some slimming of slim html

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -51,8 +51,8 @@ GEM
       rack (>= 1.0.0)
       rack-test (>= 0.5.4)
       xpath (~> 2.0)
-    capybara-webkit (1.3.1)
-      capybara (>= 2.0.2, < 2.5.0)
+    capybara-webkit (1.5.1)
```
Comment (awong-dev) [inline Gemfile.lock]: Hmm...did you do a bundle update?

This is fine, but normally I wouldn't expect to see Gemfile.lock changes.

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -71,7 +71,7 @@ html
 
       / Main
       .container
-          button.btn.btn-success.pull-right onclick="show_unicorn();" Rainbows and Unicorns
+          / button.btn.btn-success.pull-right onclick="show_unicorn();" Rainbows and Unicorns
```
Comment (awong-dev) [inline app/views/layouts/application.html.slim]: FYI, we can kill this Rainbows and Unicorns thing....or make it smaller. It's kinda getting old as a joke.

Comment (awong-dev) [inline app/views/layouts/application.html.slim]: Remove instead of commenting out?

**Comment by awong-dev** [comment]
One small nit, but otherwise LGTM

Merge away after fixing!

Also, please add something like "fixes #someisuenumber" for commit messages when possible.

**Comment by huttj** [comment]
Was there something wrong with having the patient info on the right side?

It isn't primary information for the page--the forms are--so putting it on the left impairs the page flow. The user's eyes naturally move along an F-shaped path, so it makes the most sense to put the labels of the form in that path. Otherwise, they're constantly re-tracking to find their place.

I liked the grey background, too, as it made it not as eye-catching. With both the info and the forms white, they're competing for the user's attention.

**Comment by huttj** [comment]
Also, the accordion is a great touch. It would be nice to have a +/- on the side of the text or a triangle, to indicate that it's an expandable link.

**Comment by emilyville** [comment]
@huttj nothing wrong with the right side, I had initially intended to combine it with the the adding evaluations, but then I found a better way to do that so happy to move it back to the right side : )

As for the background color, until we have a design I think it's clearest to have gray behind the headings and leave it light and clean.

**Pull request merged**